examples/pxScene2d/src/pxAnimate.h

```diff
@@ -24,6 +24,29 @@
 #include "pxConstants.h"
 class pxObject;
 struct animation;
+
+static rtString mapStatus(uint32_t status)
```

Comment:
Can we please move this function implementation into the cpp file?  We're looking to keep header files limited to definitions only with their implementations included in cpp files.